neurobooth_os/tasks/task_passage_reading.py

```diff
@@ -9,20 +9,25 @@
 
 import os.path as op
 
-from neurobooth_os.tasks.task import Task
+from neurobooth_os.tasks.task import Task_Eyetracker
 from neurobooth_os.tasks import utils
 import neurobooth_os
 
 
-class Passage_Reading(Task):
+class Passage_Reading(Task_Eyetracker):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
     def present_task(self, prompt=True, duration=0, **kwargs):
+        
+        def _update_tablet_screen_with_passage():
+            self.sendCommand('draw_box %d %d %d %d 12' % tuple([192,108,192+1536,108+864]))
+        
         fname = op.join(
             neurobooth_os.__path__[0], "tasks/assets/passage_reading_1536x864.jpg"
         )
         screen = utils.create_image_screen(self.win, fname)
+        _update_tablet_screen_with_passage()
```

Comment:
Any reason for creating the inner function above just putting it here? (Or calling a more general abstracted function like suggested before.)

Reply:
Still a WIP, will address in a different PR